lib/rspec/sidekiq/matchers/be_retryable.rb

```diff
@@ -11,7 +11,8 @@ def initialize(expected_retry)
         end
 
         def description
-          if @expected_retry.is_a?(Fixnum)
+          num_class = 1.class # Ruby 2.4 returns Integer, older returns Fixnum
+          if @expected_retry.is_a?(num_class)
```

Comment:
BigNum could just be a really big number. So if for some really odd reason you had a massive `@expected_retry` value, it would be `.class` of BigNum. The above code would handle it regardless. - I think the current code (without the Ruby 2.4 check) would have broken in that situation (however unlikely it would be to happen)